From a6855aac641d78a781ed720cf070c2b5f07bc449 Mon Sep 17 00:00:00 2001
From: Daniel Cerqueira <dan.git@lispclub.com>
Date: Tue, 20 Aug 2024 21:51:19 +0100
Subject: [PATCH] tp/Texinfo/Convert/HTML.pm: Fix EPUB validity on table border

 - fix `epubchecker` error thrown on checking for validity of an EPUB
 file. This addresses the index <table ... border=...> element and
 property.

Signed-off-by: Daniel Cerqueira <dan.git@lispclub.com>
---
 tp/Texinfo/Convert/HTML.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tp/Texinfo/Convert/HTML.pm b/tp/Texinfo/Convert/HTML.pm
index 5901e0db53..81435076d9 100644
--- a/tp/Texinfo/Convert/HTML.pm
+++ b/tp/Texinfo/Convert/HTML.pm
@@ -5978,7 +5978,7 @@ sub _convert_printindex_command($$$$)
   # now format the index entries
   $result
    .= $self->html_attribute_class('table', ["$index_name-entries-$cmdname"])
-    ." border=\"0\">\n" . '<tr><td></td>'
+    ." border=\"1\">\n" . '<tr><td></td>'
     . $self->html_attribute_class('th', ["entries-header-$cmdname"]).'>'
       # TRANSLATORS: index entries column header in index formatting
     . $self->convert_tree($self->gdt('Index Entry')) .'</th>'
-- 
2.45.2

